src/main/java/org/testng/TestNG.java

```diff
@@ -113,7 +114,7 @@
 
   /** System properties */
   public static final String SHOW_TESTNG_STACK_FRAMES = "testng.show.stack.frames";
```

Comment:
Why keeping this one here?

Reply:
Thanks for pointing that one out. It was a miss. I just grepped the codebase for `System.getProperty()` usages. Fixed this one as well.